config/config.exs

```diff
@@ -21,6 +21,8 @@ use Mix.Config
 #     config :logger, level: :info
 #
 
+config :ex_aws, :sns, region: System.get_env("AWS_DEFAULT_REGION")
```

Comment:
maybe we should pass this through the params instead.

either way, we have to document this in the readme